dist/tools/static-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (C) 2015 Lucas Jenß <lucas@x3ro.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+# Change to RIOT root
+cd "$(dirname "$0")/../../"
+
+function dep {
+    which $1 2>&1 1>/dev/null
+    if (( $? != 0 )); then
+        echo "Dependency not met: $1"
+        exit 1
+    fi
+}
+
+# Make sure all required commands are available
+dep cppcheck
+dep pcregrep
+
+
+RIOT_REMOTE_EXISTS=$(git rev-parse riot/master)
+if (( "$RIOT_REMOTE_EXISTS" != 0 )); then
+    git remote add riot https://github.com/RIOT-OS/RIOT.git
```

Comment:
Yeah, I use `upstream` as well. In my opinion, having two remotes is not as bad as not being able to run the CI scripts : But I'll come back and think about this more when I'm done with my thesis